opencv/cppbuild.sh

```diff
@@ -125,6 +125,73 @@ if [[ "$EXTENSION" == *gpu ]]; then
     GPU_FLAGS="-DWITH_CUDA=ON -DWITH_CUDNN=ON -DOPENCV_DNN_CUDA=ON -DCUDA_VERSION=11.0 -DCUDNN_VERSION=8.0 -DCUDA_ARCH_BIN='3.5' -DCUDA_ARCH_PTX='3.5' -DCUDA_NVCC_FLAGS=--expt-relaxed-constexpr -DCUDA_nppicom_LIBRARY="
 fi
 
+# exclude openblas dependencies
+if [[ "${NOOPENBLAS:-no}" == "yes" ]]; then
+    # arguments: <file name> <exclude key word>
+    function del_keyword_line {
+        fname=$1
+        ex_word=$2
+        ln=`grep -n $ex_word $fname | sed -n -e 's/:.*//gp'`
+        cnt=0
+        for l in $ln; do
+            sed -i -e "`expr ${l} - ${cnt}`d" $fname
```

Comment:
Should probably use the sedinplace function defined [here](https://github.com/bytedeco/javacpp-presets/blob/master/cppbuild.sh#L158) due to `sed -i`'s arguments not being consistent across implementations.